llvm/lib/Target/AMDGPU/SIInstrInfo.cpp

```diff
@@ -3941,19 +3996,22 @@ MachineInstr *SIInstrInfo::convertToThreeAddress(MachineInstr &MI,
     return MIB;
   }
 
-  assert(
-      Opc != AMDGPU::V_FMAC_F16_fake16_e32 &&
-      "V_FMAC_F16_fake16_e32 is not supported and not expected to be present "
-      "pre-RA");
+  assert(Opc != AMDGPU::V_FMAC_F16_t16_e32 &&
+         Opc != AMDGPU::V_FMAC_F16_fake16_e32 &&
+         "V_FMAC_F16_t16/fake16_e32 is not supported and not expected to be "
+         "present "
+         "pre-RA");
```

Comment:
```suggestion
         "V_FMAC_F16_t16/fake16_e32 is not supported and not expected to be present pre-RA");
```

Unnecessary literal break

Reply:
I don't think it handles merge or split of string literals